src/cmd/run.js

```diff
@@ -290,13 +291,14 @@ export default async function run(
     installed = true;
   }
 
-  const runningFirefox = await runner.run(profile);
+  const remotePort = await runner.findPort();
+  const runningFirefox = await runner.run(profile, remotePort);
```

Comment:
after the change I mentioned below, this could be:

````js
const { firefox: runningFirefox, port: remotePort } = await runner.run(profile, remotePort);
````